### DISCUSSION TITLE
Imports and Lookups via CLI Can't Validate/Deploy

### DISCUSSION THREAD

**Discussion by DaveReif** [speechly/speechly#171] [Q&A] [answered]
When following the documentation for [Imports and Lookups](https://docs.speechly.com/slu-examples/imports-and-lookups/), I have found that following the documented pattern for lookups as outlined [here](https://docs.speechly.com/slu-examples/imports-and-lookups/#the-lookup-entity-data-type) results in an error when the CLI `speechly validate` command is run.

The error is `Validate failed: rpc error: code = Unknown desc = Exception calling application: local variable 'app_id' referenced before assignment`

Is there some other approach to use for lookups?

**Comment by langma** [comment, marked as answer]
You found a problem with our API, which is now fixed. Basically the real problem with the configuration was hidden because of this problem, so you should now see the real problem after running `speechly validate`.

Thanks for asking!
